syntax = "proto3";
package tennis;

// Service
service predictFallPosition {
  rpc predictFallPositionController(predictFallPositionControllerRequest) returns (predictFallPositionControllerResponse) {}
}

// Request Name
message predictFallPositionControllerRequest {
  bytes points=1;
}

// Response Array
message predictFallPositionControllerResponse{
  bytes points=1;
}